tools/dynamo/gb_id_mapping.py

```diff
@@ -2,11 +2,22 @@
 
 import argparse
 import ast
-import json
+import os
 import re
+import subprocess
 import sys
+import tempfile
 from pathlib import Path
 
+from ruamel.yaml import YAML
```

Comment:
Why are we using `raumel`? PyTorch already has a dependency on `pyyaml` - why can't we use that?